### PR TITLE
chore: Removes ProjectIDGlobal

### PR DESCRIPTION
**Pull request by lantoli**
## Description

Removes ProjectIDGlobal. Now that acc and mig test are run together, there is no benefit in having global projects for testing.

Fixes network flaky test that was introduce in folowed-up PR.

This PR is a follow-up for: https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2156

Link to any related issue(s): CLOUDP-244186

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue). Please, add the "bug" label to the PR.
- [ ] New feature (non-breaking change which adds functionality). Please, add the "enhancement" label to the PR. A migration guide must be created or updated if the new feature will go in a major version.
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected). Please, add the "breaking change" label to the PR. A migration guide must be created or updated.
- [ ] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have read the [contributing guides](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/contributing/README.md)
- [x] I have checked that this change does not generate any credentials and that **they are NOT accidentally logged anywhere**.
- [x] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run make fmt and formatted my code
- [x] If changes include deprecations or removals, I defined an isolated PR with a relevant title as it will be used in the auto-generated changelog.
- [x] If changes include removal or addition of 3rd party GitHub actions, I updated our internal document. Reach out to the APIx Integration slack channel to get access to the internal document.

### CODE REVIEW

File: internal/service/networkcontainer/resource_network_container_test.go [hunk 1]
```diff
@@ -136,7 +136,8 @@ func TestAccNetworkContainer_updateIndividualFields(t *testing.T) {
 		regionUpdated    = "EU_WEST_2"
 	)
 
-	resource.ParallelTest(t, resource.TestCase{
+	// Serial so it doesn't conflict with TestAccNetworkContainer_basicAWS
+	resource.Test(t, resource.TestCase{
```
Comment (AgustinBettati) [inline internal/service/networkcontainer/resource_network_container_test.go:140]: I understand this is not strictly related to this PR, but to fix potentially flaky from nightly run right?

**Review by AgustinBettati — APPROVED** [review]
LGTM

**Review by EspenAlbert — APPROVED** [review]
LGTM

**Review by oarbusi — APPROVED** [review]

File: internal/service/networkcontainer/resource_network_container_test.go (hunk 1 above)
Comment (lantoli) [reply internal/service/networkcontainer/resource_network_container_test.go:140]: you're right, it's more related to the PR this is following-up, i'll update the description

**Pull request merged**